SetAwsCredentials.ps1

```diff
@@ -32,26 +32,32 @@ function GetAwsAccessToken()
 $awsConfigFilePath = "C:\Users\${env:UserName}\.aws\config"
 $fileContent = Get-Content $awsConfigFilePath
 $text = [string]::Join("`n", $fileContent)
-$profileMatches = [regex]::Matches($text, '(\[profile (.*))]((.|\n)*?)(sso_region = (.*))((.|\n)*?)(sso_account_id = (.*))((.|\n)*?)(sso_role_name = (.*))')
+$profileMatches = [regex]::Matches($text, '(\[profile (.*))]((.|\n)*?)(sso_region = (.*))((.|\n)*?)(sso_account_id = (.*))((.|\n)*?)(sso_role_name = (.*))((.|\n)*?)(region = (.*))')
 $profiles=@{}
-$regions=@{}
+$sso_regions=@{}
 $accounts=@{}
 $roles=@{}
-Foreach($profileMatch in ($profileMatches | Sort-Object -Property Value)) {
+$regions=@{}
+$count = 0
+Foreach($profileMatch in $profileMatches) {
 	$profile = $profileMatch.Groups[2].Value
-	$profiles.Add($profile, $profile)
-	$region = $profileMatch.Groups[6].Value
-	$regions.Add($profile, $region)
+	$profiles.Add($count, $profile)
```

Comment:
[0, plant-sandbox], [1, plant-non-prod]...this way we can use a custom sort order from the config file..
I prefer sandbox, non-prod, prod order, which is non-alphabetically.